spec/models/miq_server/configuration_management_spec.rb

```diff
@@ -73,11 +73,11 @@
       end
 
       it "does not overwrite the servers zone_id if the config value is invalid" do
-        _guid, miq_server, zone = EvmSpecHelper.local_guid_miq_server_zone
+        miq_server = EvmSpecHelper.local_miq_server
```

Comment:
This one should get the zone _before_ the next call.  This will ensure that the "previous" zone was not modified.  The way it is now, the .zone method is not executed until afterwards, which may not reflect the intentions of the test correctly.